docs/control-structures.rst

```diff
@@ -10,7 +10,7 @@ Control Structures
 Most of the control structures from C/JavaScript are available in Solidity
 except for ``switch`` and ``goto``. So
 there is: ``if``, ``else``, ``while``, ``for``, ``break``, ``continue``, ``return``, ``? :``, with
-the usual semantics known from C / JavaScript.
+the usual semantics known from C/JavaScript.
```

Comment:
I think it is fine like this. You can change it to `C or JavaScript` if you prefer that.